language/diem-tools/writeset-transaction-generator/src/main.rs

```diff
@@ -114,7 +104,7 @@ fn main() -> Result<()> {
             version,
             first_release,
         } => {
-            let release_modules = stdlib_modules();
+            let release_modules = diem_framework_modules();
```

Comment:
We should probably use the new create_release api here. Feel free to leave it as is and I can modify this logic in another PR.

Reply:
Conceptually agreed, but it's not clear to me how that will work out since `create_release` outputs to files instead of returning something back in Rust.

Reply:
Feel free to keep it as is. I will integrate the logic in later PR.